src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -522,7 +522,7 @@ def export(
             task (Optional[str]): The task to use for exporting. Defaults to None.
             codebase (Optional[str]): The codebase to use for exporting. Defaults to None.
             export_type (str): The type of export to perform. Can be "ONNX" or "OPENVINO". Defaults to "OPENVINO".
-            deploy_config (Optional[str]): The path to the deploy config file to use for exporting. Defaults to None.
+            deploy_config (Optional[str, dict]): The path to the deploy config to use for exporting. Defaults to None.
```

Comment:
Thanks for the comment ! We already used `ABC (Optional[...])` or `ABC (..., optional)` format instead of `ABC ((... | ... | None))` over the entire code. If we'll use the new convention for the docstring, how about changing entire things at the same time at the further separated PR?

Reply:
yeah, that's fine. As you said, the docstrings seems to be following the old convention. We need to ensure that all of the docstring follow the new convention